packages/nhsuk-frontend/src/nhsuk/core/styles/_icons.scss

```diff
@@ -1,60 +1,18 @@
 @use "../settings" as *;
+@use "../tools" as *;
 
 ////
 /// Icons
 ///
 /// @group styles
 ////
 
-/// Default icon size
+/// Default icon styles
 
 .nhsuk-icon {
-  height: $nhsuk-icon-size;
-  width: $nhsuk-icon-size;
-}
-
-/// Default icon colours
-
-.nhsuk-icon__search {
-  fill: currentcolor;
-}
-
-.nhsuk-icon__user {
   fill: currentcolor;
 }
 
-.nhsuk-icon__cross {
-  fill: $color_nhsuk-red;
-}
-
-.nhsuk-icon__tick {
-  stroke: $color_nhsuk-green;
-}
-
-.nhsuk-icon__arrow-right {
-  fill: $color_nhsuk-blue;
-}
-
-.nhsuk-icon__arrow-left {
-  fill: $color_nhsuk-blue;
-}
-
-.nhsuk-icon__arrow-right-circle {
-  fill: $color_nhsuk-green;
-}
-
-.nhsuk-icon__chevron-down {
-  fill: currentcolor;
-}
-
-.nhsuk-icon__plus {
-  fill: $color_nhsuk-blue;
-}
-
-.nhsuk-icon__minus {
-  fill: $color_nhsuk-blue;
-}
-
 /// Icon size adjustments
 
 .nhsuk-icon--size-25 {
```

Comment:
Can we remove these icon size classes? They don’t seem to be used anywhere, or documented in the service manual.